website/docs/r/cloud_provider_access.markdown

```diff
@@ -77,6 +73,8 @@ resource "mongodbatlas_cloud_provider_access_setup" "test_role" {
 * `last_updated_date`                - Date and time when this Azure Service Principal was last updated. This parameter expresses its value in the ISO 8601 timestamp format in UTC.
 * `role_id`                        - Unique ID of this role.
 
+* -> **NOTE:** the two sub-attributes of the aws_config block will likely be needed on the external AWS account side as the constraints on the trust policy of an IAM role you're granting Atlas access to. They can be accessed via `mongodbatlas_cloud_provider_access_setup.<role>`, for example the trust policy will reference `mongodbatlas_cloud_provider_access_setup.<role>.aws_config[0].atlas_aws_account_arn` in its `Principal` statement, and the `mongodbatlas_cloud_provider_access_setup.<role>.aws_config[0].atlas_assumed_role_external_id` in its `Condition.StringEquals.sts:ExternalId` statement.
```

Comment:
We have an AWS example which helps understand the usage of both attributes: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/examples/atlas-cloud-provider-access/aws/aws-roles.tf#L19. Curious if you had seen this example, and if this note is worth adding having this defined.

Reply:
Yes agree the examples are not very accessible from our docs. I believe we could add a link to the example I referenced within the `Example Usage with AWS` section. cc: @Zuhairahmed

Reply:
Got it - I've changed the note to a link